src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/services/TaskManagementServiceTest.java

```diff
@@ -245,17 +263,111 @@ void cancelTask_should_rollback_transaction_when_exception_occurs_calling_camund
             doThrow(FeignException.FeignServerException.class)
                 .when(camundaServiceApi).bpmnEscalation(any(), any(), any());
 
-            createAndSaveTestTask(taskId);
+            createAndSaveTestTask(taskId, UNCONFIGURED);
+
+            assertThatThrownBy(() -> transactionHelper.doInNewTransaction(
+                () -> taskManagementService.cancelTask(taskId, accessControlResponse)))
+                .isInstanceOf(TaskCancelException.class)
+                .hasNoCause()
+                .hasMessage("Task Cancel Error: Unable to cancel the task.");
+
+            verifyTransactionWasRolledBack(taskId, UNCONFIGURED);
+
+        }
+
+        @Test
+        void should_set_task_state_terminated_when_camunda_api_throws_an_exception_and_cft_task_state_is_assigned() {
```

Comment:
will this be the same expectation for unassigned?

Reply:
sure